core/templates/dev/head/pages/error-pages/error-page.controller.ts

```diff
@@ -30,17 +30,20 @@ angular.module('oppia').directive('errorPage', [
         '/pages/error-pages/error-page.directive.html'),
       controllerAs: '$ctrl',
       controller: [
-        'PageTitleService', 'UrlInterpolationService',
+        '$http', 'PageTitleService', 'UrlInterpolationService',
         function(
-            PageTitleService, UrlInterpolationService) {
+            $http, PageTitleService, UrlInterpolationService) {
           var ctrl = this;
           ctrl.oopsMintImgUrl = UrlInterpolationService.getStaticImageUrl(
             '/general/oops_mint.png');
 
-          ctrl.statusCode = GLOBALS.status_code;
-
-          PageTitleService.setPageTitle(
-            'Error ' + ctrl.statusCode + ' - Oppia');
+          $http.get(document.location.href, { observer: 'response' }).error(
```

Comment:
Do we need to make a second request to retrieve the status? Can we somehow try to retrieve the status code from the first request?

Reply:
It is possible, but we need to hack the `UserAgent` content returned by the server.  https://stackoverflow.com/a/33829607/7676306

Reply:
I don't understand what you mean by the link you provided.

Reply:
Sorry pasted a wrong link https://stackoverflow.com/a/220233/7676306

Reply:
Okay, I don't really like making the same request again, because it can cause some other problems. But I'm not sure if there is any other solution that doesn't require Jinja.